docker-app/worker_wrapper/wrapper.py

```diff
@@ -264,15 +270,31 @@ def _run_docker(
             detach=True,
             mem_limit=config.WORKER_QGIS_MEMORY_LIMIT,
             cpu_shares=config.WORKER_QGIS_CPU_SHARES,
+            labels={
+                "app": "worker",
+                "type": self.job.type,
+                "job_id": str(self.job.id),
+                "project_id": str(self.job.project_id),
```

Comment:
@suricactus added this on your request. Goood?

Reply:
Lovely!